pyproject.toml

```diff
@@ -24,5 +24,19 @@ dependencies = [
   'uvloop == 0.16.0; platform_machine == "x86_64"',
   'uvloop == 0.16.0; platform_machine == "i686"'
 ]
+classifiers = [
+        "Development Status :: 3 - Alpha",
+        "Environment :: Web Environment",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: OS Independent",
+        "Topic :: Internet :: WWW/HTTP",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: Implementation :: CPython",
+        "Programming Language :: Python :: Implementation :: PyPy",
```

Comment:
Does it really supports PyPy? :thinking:

Reply:
Oh, right. It doesn't. I misread it as `PyPi`.

Thank you! :D

Reply:
Updated.